openxcframework/BluetoothManager.swift

```diff
@@ -39,27 +39,27 @@ open class BluetoothManager: NSObject,CBCentralManagerDelegate,CBPeripheralDeleg
   // config for outputting debug messages to console
   fileprivate var managerDebug : Bool = false
   var result: String!
-  var troughputtimer: Timer!
+  var troughputTimer: Timer!
```

Comment:
The variable name should be throughputTimer

Reply:
done